crates/holochain/CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Prevent duplicate calls to `init`. Previously, calling `init` directly would result in 2 calls to `init` if the zome had not been initialised yet and 1 call if the zome had been initialised. Now, calling `init` directly will result in 1 call by the conductor to initialise the zome and all subsequent calls to `init` will return `InitCallbackResult::Pass`. This both fixes surprising behavior and allows `init` to be called directly to initialise a zome if desired.
 
 ## 0.5.0-dev.10
-
+  - Added LinkTag helper trait functions to go into and out with serialized bytes
```

Comment:
This needs to be under `Unreleased` further up

Reply:
This looks like it's now under the `## 0.5.0-dev.11`? It needs to go right up to the `Unreleased` heading